include/cuco/detail/open_addressing/open_addressing_impl.cuh

```diff
@@ -365,6 +410,48 @@ class open_addressing_impl {
         first, num_keys, stencil, pred, container_ref);
   }
 
+  /**
+   * @brief Asynchronously erases keys in the range `[first, last)`.
+   *
+   * For each key `k` in `[first, last)`, if contains(k) returns true, removes `k` and it's
+   * associated value from the container. Else, no effect.
+   *
+   *  Side-effects:
+   *  - `contains(k) == false`
+   *  - `find(k) == end()`
+   *  - `insert({k,v}) == true`
+   *  - `size()` is reduced by the total number of erased keys
```

Comment:
@sleeepyjack how to properly @note in this case?

Reply:
Like this? https://stackoverflow.com/a/34016569

No clue how it looks after rendering.